### ISSUE TITLE
OkHttpClient connection was leaked warning

### ISSUE THREAD

**Issue by z4kn4fein** [configcat/java-sdk#6]
Investigate this warning:

```
2021-01-10 23:25:45.108 WARN 61 --- [ ConnectionPool] okhttp3.OkHttpClient : A connection to https://cdn-global.configcat.com/ was leaked. Did you forget to close a response body? To see where this was allocated, set the OkHttpClient logger level to FINE: Logger.getLogger(OkHttpClient.class.getName()).setLevel(Level.FINE);
```

**Comment by z4kn4fein** [comment]
It seems the response body must be closed every time, regardless of the response status code. More info: https://github.com/square/okhttp/issues/2311

**Comment by z4kn4fein** [comment]
Fix released in [v6.0.1](https://github.com/configcat/java-sdk/releases/tag/v6.0.1)
